debian/mariadb-client-10.5.links

```diff
@@ -1,21 +1,20 @@
-usr/bin/mysql_find_rows usr/bin/mariadb-find-rows
-usr/bin/mysql_fix_extensions usr/bin/mariadb-fix-extensions
-usr/bin/mysql_plugin usr/bin/mariadb-plugin
-usr/bin/mysql_waitpid usr/bin/mariadb-waitpid
-usr/bin/mysqlaccess usr/bin/mariadb-access
-usr/bin/mysqladmin usr/bin/mariadb-admin
-usr/bin/mysqlbinlog usr/bin/mariadb-binlog
-usr/bin/mysqlcheck usr/bin/mariadb-analyze
-usr/bin/mysqlcheck usr/bin/mariadb-optimize
-usr/bin/mysqlcheck usr/bin/mariadb-repair
-usr/bin/mysqlcheck usr/bin/mysqlanalyze
-usr/bin/mysqlcheck usr/bin/mysqloptimize
-usr/bin/mysqlcheck usr/bin/mysqlrepair
-usr/bin/mysqldump usr/bin/mariadb-dump
-usr/bin/mysqldumpslow usr/bin/mariadb-dumpslow
-usr/bin/mysqlimport usr/bin/mariadb-import
-usr/bin/mysqlreport usr/bin/mariadb-report
-usr/bin/mysqlslap usr/bin/mariadb-slap
+usr/bin/mariadb-find-rows usr/bin/mysql_find_rows
+usr/bin/mariadb-fix-extensions usr/bin/mysql_fix_extensions
+usr/bin/mariadb-waitpid usr/bin/mysql_waitpid
+usr/bin/mariadb-access usr/bin/mysqlaccess
+usr/bin/mariadb-admin usr/bin/mysqladmin
+usr/bin/mariadb-check usr/bin/mariadb-analyze
+usr/bin/mariadb-check usr/bin/mariadb-optimize
+usr/bin/mariadb-check usr/bin/mariadb-repair
+usr/bin/mariadb-check usr/bin/mysqlanalyze
+usr/bin/mariadb-check usr/bin/mysqloptimize
+usr/bin/mariadb-check usr/bin/mysqlrepair
+usr/bin/mariadb-dump usr/bin/mysqldump
+usr/bin/mariadb-dumpslow usr/bin/mysqldumpslow
+usr/bin/mariadb-import usr/bin/mysqlimport
+usr/bin/mariadb-report usr/bin/mysqlreport
+usr/bin/mariadb-slap usr/bin/mysqlslap
+
```

Comment:
empty line, this as well would be automatically fixed by wrap-and-sort

Reply:
I ran wrap-and-sort -a -v in new commit cf70893